CONTRIBUTION_GUIDE.md

```diff
@@ -0,0 +1,69 @@
+### Choosing Stories/tickets.
+
+When deciding on an issue to work on, look for the  `Help Wanted` or `Good First Issue` tags.
+
+### Branch Naming Conventions
+
+Kindly follow these conventions when creating branches and pull requests.
+```
+git checkout develop
+```
+``` 
+git pull upstream develop
+```
+
+Ensure you have setup AgileVentures/WebsiteOne-FE's upstream  `develop`.  Otherwise you will not have the latest `develop ` changes.
+
+To confirm this, run ```git remote -v```.
+You should see a simillar output.
+```
+origin  https://github.com/yourgithubusername/WebsiteOne-FE.git (fetch)
+origin  https://github.com/yourgithubusername/WebsiteOne-FE.git (push)
+upstream    https://github.com/AgileVentures/WebsiteOne-FE (fetch)
+upstream    https://github.com/AgileVentures/WebsiteOne-FE (push)
+```
+
+If not, you need to set the remote develop in order to get the latest copy once changes are merged.
+
+Inorder to achieve that, run:
+
+```
+git remote add upstream https://github.com/AgileVentures/WebsiteOne-FE
+```
+
+Then run 
+```
+git pull upstream develop
+``` 
+depending on the name of your origin.
+
+You will now have the latest copy of develop in your local.
+
+After that run 
+```
+git merge upstream develop 
```

Comment:
@Felistas Would you mind explaining the purpose of this command `git merge upstream develop`? I think it might end up confusing newcomers, as it has confused me. Might it not be necessary, since we already ran `git pull upstream develop`?

Reply:
@FedericoEsparza I don't think its necessary.You are right.